source/guides/clear/autospec.rst

```diff
@@ -326,6 +326,90 @@ and dependencies so autospec can complete a successful build.
    if autospec is not able to automatically find the correct dependency on its
    own.
 
+.. TODO: Document how to set up a license server for use with autospec
+
+Example 5: Control file management
+**********************************
+
+TBD
+
+Example 6: Update an existing package
+*************************************
+
+Generally, developers can expect a more streamlined approach to managing a large collection of packages with :command:`autospec`.
+
+The |CL| team prefers to carry no patches and seeks to make the latest releases work. If we do need patches, we use :command:`autospec` to add, remove, or manage patches. The :command:`autospec` control files are integral to the patch management process.
+
+Adding and submitting patches
+=============================
+
+* To add patches to |CL| upstream, follow our process
+  in the |CL| `distribution repo contributing guide`_.
+
+* To submit a patch to upstream, follow our process in
+  `contributing new packages`_.
```

Comment:
For *submitting patches*, in addition to the above link, I think it will help to link to the section for contributing patches to existing packages as well: https://github.com/clearlinux/distribution/blob/master/contributing.md#contributing-to-an-existing-software-package